cpp/ps0007/problem_s_8.cpp

```diff
@@ -26,10 +26,13 @@ float calculateArea(string shape, unsigned params[]) {
     }
 }
 int main() {
+    // prompting user for the shape 1/1.5
     string shape;
     unsigned paramCount;
     cout << "Enter the shape: ";
     cin >> shape;
```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input
-0.5 mark cuz of your not clear prompt (make it easy for the user to enter only the first character or choose a number from a list